python-sdk/nuscenes/eval/detection/README.md

```diff
@@ -65,14 +65,14 @@ Note that the previous [evaluation server](https://eval.ai/web/challenges/challe
 A summary of the results can be seen below. 
 For details, please refer to the [detection leaderboard](https://www.nuscenes.org/object-detection).
 
-| Rank | Team name          | NDS   |
-|---   |---                 |---    |
-|  1   | Noah CV Lab fusion | 69.7% |
-|  2   | CenterPoint        | 67.5% |
-|  3   | CVCNet ensemble    | 66.6% |
-|  4   | PanoNet3D          | 63.1% |
-|  5   | CRIPAC             | 63.2% |
-|  6   | SSN                | 61.6% |
+| Rank | Team name          | NDS   | Award           |
+|---   |---                 |---    |---              |
+|  1   | Noah CV Lab fusion | 69.7% | Best submission |
```

Comment:
a typo here? This should be 69.0%

Reply:
Yes, I confused it with the other Noah submission. Thanks for spotting this.
It will be addressed in https://github.com/nutonomy/nuscenes-devkit/pull/535.